src/main/java/seedu/address/logic/parser/CsvAttendanceParser.java

```diff
@@ -0,0 +1,32 @@
+package seedu.address.logic.parser;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import seedu.address.logic.parser.exceptions.ParseException;
+/**
+ * Parses attendance data from a CSV field, extracting the weeks present as a set of integers.
+ */
+public class CsvAttendanceParser {
+    /**
+     * Parses the weeksPresent field into a set of integers.
+     *
+     * @param weeksField string representing weeks present, separated by commas
```

Comment:
Not sure if this is very important, but the java coding standard specifies a punctuation behind each param field. I think most of the comments lacks the punctuation.